backend/src/main/java/com/daedan/festabook/place/controller/PlaceController.java

```diff
@@ -27,6 +31,32 @@ public class PlaceController {
     private final PlaceService placeService;
     private final PlacePreviewService placePreviewService;
 
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    @Operation(summary = "특정 조직의 세부 사항 없이 플레이스 저장")
```

Comment:
```suggestion
    @Operation(summary = "특정 조직의 세부 사항 없이 플레이스 생성")
```

어때요? 저장은 save 느낌이 있어보여요!